mailbox/store/src/test/java/org/apache/james/mailbox/store/search/AbstractMessageSearchIndexTest.java

```diff
@@ -852,18 +852,6 @@ void addressShouldReturnUidHavingRightExpeditorWhenFromIsSpecifiedWithOnlyUserPa
             .containsOnly(m8.getUid());
     }
 
-    @Test
-    void addressShouldReturnUidHavingRightExpeditorWhenFromIsSpecifiedWithDomainPartOfEmail() throws Exception {
```

Comment:
Shouldn't we just `@Disabled("JAMES-3406 Partial address matches are not supported")` ?

Same thing for all tests here.